src/components/organisms/ProfileModal/EditProfileForm/EditProfileForm.tsx

```diff
@@ -68,64 +75,69 @@ const EditProfileForm: React.FunctionComponent<PropsType> = ({
   const pictureUrl = watch("pictureUrl");
 
   return (
-    <div className="edit-profile-modal">
-      <h1 className="title">Edit profile</h1>
-      <form onSubmit={handleSubmit(onSubmit)} className="form">
-        <div className="input-group profile-form">
-          <input
-            name="partyName"
-            className="input-block input-centered"
-            placeholder="Your display name"
-            ref={register({
-              required: true,
-              maxLength: DISPLAY_NAME_MAX_CHAR_COUNT,
-            })}
+    <div className="EditProfileForm">
+      <h1 className="EditProfileForm__title">Edit profile</h1>
+      <div className="form">
```

Comment:
should it have onSubmit?

Reply:
Should it be within a proper `form` tag?

Reply:
added `onSubmit`, `form`-tag: https://github.com/sparkletown/sparkle/commit/4b9c4fee3c3c8ac3d75c4a18814995447bfa29da